### PR TITLE
update jsoup to 1.11.3

### PR DESCRIPTION
**Pull request by maximermilov**
1.11.3 contains a lot of bug fixes.
including https://github.com/jhy/jsoup/pull/901,
which allows using templates inside table.

### CODE REVIEW

File: closure/repositories.bzl [hunk 1]
```diff
@@ -888,10 +888,9 @@ def org_jsoup():
         name = "org_jsoup",
         licenses = ["notice"],  # The MIT License
         jar_urls = [
-            "https://mirror.bazel.build/repo1.maven.org/maven2/org/jsoup/jsoup/1.10.2/jsoup-1.10.2.jar",
-            "https://repo1.maven.org/maven2/org/jsoup/jsoup/1.10.2/jsoup-1.10.2.jar",
+            "https://repo1.maven.org/maven2/org/jsoup/jsoup/1.11.3/jsoup-1.11.3.jar",
```
Comment (gkdn) [inline closure/repositories.bzl]: @dslomov rules_closure generally provides at least 2 URLs including bazel.build ones. What's the recommended policy here?

Comment (maximermilov) [reply closure/repositories.bzl]: I am not sure how to upload into mirror.bazel.build

Comment (maximermilov) [reply closure/repositories.bzl]: Friendly ping

**Comment by dslomov** [comment]
Ouch sorry for dropping this... Uploaded to `https://mirror.bazel.build/repo1.maven.org/maven2/org/jsoup/jsoup/1.11.3/jsoup-1.11.3.jar`

**Pull request merged**